moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -1489,7 +1493,7 @@ bool TrajectoryExecutionManager::executePart(std::size_t part_index)
       if (context.trajectory_parts_[longest_part].joint_trajectory.points.size() >=
           context.trajectory_parts_[longest_part].multi_dof_joint_trajectory.points.size())
       {
-        rclcpp::Duration d(0.0);
+        rclcpp::Duration d(std::chrono::seconds(0));
```

Comment:
In this case I would do this:

```suggestion
        auto d = rclcpp::Duration::from_seconds(0);
```